README.md

```diff
@@ -19,32 +21,33 @@ See also other Orbit clients:
 - [orbit-web](https://github.com/orbitdb/orbit-web) - Browser App
 - [orbit-electron](https://github.com/orbitdb/orbit-electron) - Desktop App
 
-
 ## Run
 
 Get the source code and install dependencies:
 
-```
+```sh
 git clone https://github.com/orbitdb/orbit-textui.git
 cd orbit-textui
 npm install
 ```
 
 Start the program:
 
-```
+```sh
 npm start
 ```
 
 Or instead of `npm start`, run the following to specify your chat name:
 
-```
+```sh
 node index <nickname>
 ```
 
 ## Contribute
 
-Contributions are welcome! Please open [an issue](https://github.com/orbitdb/orbit-textui/issues) if there is something you would like to see.
+If you think this could be better, please [open an issue](https://github.com/orbitdb/orbit-textui/issues/new)!
+
+Please note that all interactions in [@orbitdb](https://github.com/orbitdb) fall under our [Code of Conduct](CODE_OF_CONDUCT.md).
```

Comment:
Can you add the full Contribute section from the repo-template?

Reply:
Done!